### PR TITLE
Sp2 api style check

### PR DESCRIPTION
**Pull request by Spellbreeze**
Contributor(s): Alexander Goebel

### CODE REVIEW

File: backend/controllers/mail_controller.js [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+//    subject: subject,
+//    text: plaintext,
+//    html: html
+//  };
+//
+//  transport.sendMail(message, (error, info) => {
+//    if (error) {
+//    return console.log(error);
+//    }
+//    console.log('Message sent: %s', info.messageId);
```
Comment (madhura112) [inline backend/controllers/mail_controller.js:25]: Don't need this

File: backend/controllers/mail_controller.js [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
-    return;
-        
-    var message = {
-        from: '"Civility Optics" <no-reply@civilityoptics.com>',
-        to: recipient,
-        subject: subject,
-        text: plaintext,
-        html: html
-    };
+//  return
```
Comment (madhura112) [inline backend/controllers/mail_controller.js]: Can remove this as well

**Review by madhura112 — COMMENTED** [review]
PR looks good to me, but let's not merge until after testing to make sure the linter didn't introduce any bugs

File: backend/controllers/mail_controller.js [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
 
-    transport.sendMail(message, (error, info) => {
-        if (error) {
-        return console.log(error);
-        }
-        console.log('Message sent: %s', info.messageId);
-    })
+//  var message = {
```
Comment (parkaransingh) [inline backend/controllers/mail_controller.js:13]: Remove comments

**Review by parkaransingh — APPROVED** [review]
Need to test with es lint locally before merging. Hold on the merge. Otherwise looks good.

**Pull request merged**